src/leaflet/core.cljs

```diff
@@ -67,15 +67,15 @@
       (.on marker "mouseout" #(mouseout-fn marker % point)))
     marker))
 
-(defn create-point [point {:keys [lat-fn lon-fn style-fn popup-fn mouseover-fn mouseout-fn], :or {lat-fn :lat, lon-fn :lon}, :as props}]
+(defn create-point [point {:keys [lat-fn lon-fn style-fn icon-fn popup-fn mouseover-fn mouseout-fn shape], :or {lat-fn :lat, lon-fn :lon}, :as props}]
```

Comment:
I think it'd be preferable to use a `:marker-layer` instead of changing the elements for the `:point-layer`. Marker layer already has an `icon-fn` parameter even.